tests/integration/linux_bridge_test.yaml

```diff
@@ -0,0 +1,248 @@
+    - name: eth2
+      stp-hairpin-mode: false
+      stp-path-cost: 100
+      stp-priority: 32
+description: Create the bond interface testbond0 with the controller linux-br0, the bonding mode balance-rr, and ethernet port eth1
+interfaces:
+- name: testbond0
+  type: bond
+  state: up
+  controller: linux-br0
+  link-aggregation:
+    mode: balance-rr
+    port:
+    - eth1
+description: Configure the linux bridge linux-br0 with the group address 01:80:C2:00:00:04
+interfaces:
+- name: linux-br0
+  type: linux-bridge
+  state: up
+  bridge:
+    options:
+      group-addr: 01:80:C2:00:00:04
+description: Configure the linux bridge linux-br0 with multicast-snooping set to true
+interfaces:
+- name: linux-br0
+  type: linux-bridge
+  state: up
+  bridge:
+    options:
+      multicast-snooping: true
+description: Set the controller of ethernet device eth1 to empty
+interfaces:
+- name: eth1
+  type: ethernet
+  state: up
+  controller: ''
+description: bring up the linux bridge linux-br0
+interfaces:
+- name: linux-br0
+  type: linux-bridge
+  state: up
+description: Configure the linux bridge linux-br0 with port eth2 attached, do not allow traffic to be sent back to the same port on which it was received, set the stp path cost to 100, set the stp priority to 32
+interfaces:
+- name: linux-br0
+  type: linux-bridge
+  state: up
+  bridge:
+    port:
+    - name: eth2
+      stp-hairpin-mode: false
+      stp-path-cost: 100
+      stp-priority: 32
+description: Create bridge linux-br0, add the ethernet port eth1 and eth2 to the linux bridge. In ethernet port eth1, do not allow traffic to be sent back to the same port on which it was received, set stp path cost to 100, set stp priority to 32.
+interfaces:
+- name: linux-br0
+  type: linux-bridge
+  state: up
+  bridge:
+    port:
+    - name: eth1
+      stp-hairpin-mode: false
+      stp-path-cost: 100
+      stp-priority: 32
+    - name: eth2
+description: Set accepting all mac to false for linux bridge linux-br0
+interfaces:
+- name: linux-br0
+  type: linux-bridge
+  state: up
+  accept-all-mac-addresses: false
+description: Set accepting all mac to true for linux bridge linux-br0
+interfaces:
+- name: linux-br0
+  type: linux-bridge
+  state: up
+  accept-all-mac-addresses: true
+description: Configure the linux bridge linux-br0 with the multi-cast router disabled
+interfaces:
+- name: linux-br0
+  type: linux-bridge
+  state: up
+  bridge:
+    options:
+      multicast-router: disabled
+description: Configure the linux bridge linux-br0 with the multi-cast router 0
+interfaces:
+- name: linux-br0
+  type: linux-bridge
+  state: up
+  bridge:
+    options:
+      multicast-router: disabled
+description: Configure the linux bridge linux-br0 with the multi-cast router enabled
+interfaces:
+- name: linux-br0
+  type: linux-bridge
+  state: up
+  bridge:
+    options:
+      multicast-router: enabled
+description: Configure the linux bridge linux-br0 with the multi-cast router 2
```

Comment:
Even we desired number 2, but when generating diff, nmstate uses `multicast-router: enabled`. So I think you need to change your `apply_with_description()` there.